### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by nora99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,107 @@
 # mobile-update-check
 
-
 [![Build Status](https://travis-ci.org/egymgmbh/mobile-update-check.svg?branch=master)](https://travis-ci.org/egymgmbh/mobile-update-check)
 [![codecov](https://codecov.io/gh/egymgmbh/mobile-update-check/branch/master/graph/badge.svg)](https://codecov.io/gh/egymgmbh/mobile-update-check)
 [![Go Report Card](https://goreportcard.com/badge/github.com/egymgmbh/mobile-update-check)](https://goreportcard.com/report/github.com/egymgmbh/mobile-update-check)
 [![eGym Core Team](https://img.shields.io/badge/eGym-Core%20Team-orange.svg)](https://www.egym.com/)
+
+The Mobile-Update-Check offers a simple way to check an apps version and
```
Comment (danrl) [inline README.md]: app's

File: README.md [hunk 2]
```diff
@@ -1,7 +1,107 @@
...
+potentially request an update from the user if necessary.
+
+Currently only product and os-version numbers of the following format are supported:
+```
```
Comment (danrl) [inline README.md]: Reference to semantic versioning pls.

File: README.md [hunk 3]
```diff
@@ -1,7 +1,107 @@
...
+MAJOR.MINOR.PATCH
+```
+
+# Adding Update Rules
+In order to add a rule for an application, one needs to add the rules to the
+
+```
+rules.json
```
Comment (danrl) [inline README.md]: Inline fixed font is probably enough, `rules.json`

File: README.md [hunk 4]
```diff
@@ -1,7 +1,107 @@
...
+```
+The rules.json file contains a list of rules-sets. Each ruleset contains a list of rules for a key.
```
Comment (danrl) [inline README.md]: Rule-sets vs rulesets

File: README.md (hunk 1 above)
Comment (nora99) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (nora99) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (nora99) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (nora99) [reply README.md]: Done

**Pull request merged**